### ISSUE TITLE
[Bug]: Blank and unresponsive

### ISSUE THREAD

**Issue by eshsdhsd** [FreeTubeApp/FreeTube#4701]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

The app at time goes blank for a few seconds and it's very annoying. I don't like it and I don't know whats going on
![l](https://github.com/FreeTubeApp/FreeTube/assets/79751239/ed6fc6bb-d671-4789-b741-11c19af5d30c)

### Expected Behavior

Sometimes The app at time goes blank for a few seconds and it's very annoying. I don't like it and I don't know whats going on

It happens when reloading for opening a new window

### Issue Labels

accessibility issue, API issue, content not loading, inconsistent behavior

### FreeTube Version

v0.19.1 Beta

### Operating System Version

7

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

the versions just before

### Additional Information

It would be nice to know wether something is loading or not

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
You're saying that this is random?

**Comment by eshsdhsd** [comment]
I think with the new update it's probably gone could have been invidious because it fellback to it when the Local api failed witch is fixed now, so I think it's resolved.

**Comment by absidue** [comment]
From your screenshot you are using Windows 7, as  future releases will only support Windows 10 and above (Chromium dropped support for Windows 7, 8 and 8.1), even if we did find a way to fix it (unlikely as I can't make the issue happen on my machine and you haven't provided any steps to reproduce the issue), you wouldn't be able to you a version with the fix anyway.

**Comment by eshsdhsd** [comment]
> From your screenshot and text in the issue you are using Windows 7, as future releases will only support Windows 10 and above (Chromium dropped support for Windows 7, 8 and 8.1), even if we did find a way to fix it (unlikely as I can't make the issue happen on my machine and you haven't provided any steps to reproduce the issue), you wouldn't be able to use a version with the fix anyway.

Could you try throttle your disk and make it slow? I don't know I already had a video playing and opened a new window then it was slow, don't know if that would reproduce the issue. Maybe it's the disk?

**Comment by eshsdhsd** [comment]
> Also as this is probably the same problem as the other issue that you commented on, there is no point reopening this issue and having two open issues for the same problem (more productive if the discussion happens in one place).

Ok sorry then just close this one Ill be on the other thread
